src/iexamine.cpp

```diff
@@ -792,6 +792,57 @@ void iexamine::cardreader( player &p, const tripoint &examp )
     }
 }
 
+void iexamine::cardreader_robofac( player &p, const tripoint &examp )
+{
+    itype_id card_type = "id_science";
```

Comment:
Accepting random science cards found on random bodies in nowhere seems... wrong. Maybe add a separate - Melchior - type of science cards?

Reply:
They dont actually accept it!

They take it, but then block the doors and kindly tell you to go away.